addons/mail/static/src/core/public_web/discuss.js

```diff
@@ -89,6 +91,29 @@ export class Discuss extends Component {
             () => this.actionPanelAutoOpenFn(),
             () => [this.thread]
         );
+        useEffect(() => {
+            Object.assign(this.state, {
+                shouldDisplayUserTime: false,
+                otherUserTime: null,
+                otherUserDate: null,
+                otherUserTz: null,
+            });
+            if (!this.thread || this.thread.channel_type !== "chat") return;
```

Comment:
Enable eslint (see web/tooling). We do do these inline returns. It's not readable.